src/main/java/com/amazon/jenkins/ec2fleet/fleet/EC2CreateFleet.java

```diff
@@ -0,0 +1,169 @@
+package com.amazon.jenkins.ec2fleet.fleet;
+
+import com.amazon.jenkins.ec2fleet.FleetStateStats;
+import com.amazon.jenkins.ec2fleet.Registry;
+import com.amazonaws.services.ec2.AmazonEC2;
+import com.amazonaws.services.ec2.model.*;
+import hudson.util.ListBoxModel;
+import org.springframework.util.ObjectUtils;
+
+import java.util.*;
+
+public class EC2CreateFleet implements EC2Fleet {
```

Comment:
The product name is `EC2Fleet` like `SpotFleet`. Since `EC2Fleet` is used for the interface here, I think it should be renamed to `Fleet` and this class should be named `EC2Fleet` instead.

Reply:
On second thoughts, since many classes in the repo are named with prefix `EC2Fleet`, I would like to suggest an alternative name `EC2EC2Fleet`. What do you think?

Reply:
I prefer changing the interface name to Fleet and this class to be named EC2Fleet, but can we do that without having to change the names of every other class prefix?

Reply:
I think changing the current usage of the prefix EC2Fleet might be confusing. Let's do it all the way and rename the classes with the prefix (in a separate commit)?